modules/computation/ec2.tf

```diff
@@ -41,3 +41,15 @@ resource "aws_iam_instance_profile" "ecs_instance_role" {
   name = local.ecs_instance_role_name
   role = aws_iam_role.ecs_instance_role.name
 }
+
+resource "aws_security_group" "this" {
+  name   = local.batch_security_group_name
+  vpc_id = var.metaflow_vpc_id
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = var.compute_environment_egress_cidr_blocks
+  }
```

Comment:
I think we'd want to have a rule to allow ingress to self as well too (the Default SG includes it). Otherwise this change may break deployments for people that happen to use multi-node parallel jobs.